lib/migrations/20170627135433_food_entry.js

```diff
@@ -1,21 +1,21 @@
+'use strict';
 
-exports.up = function(knex, Promise) {
-    // What does Promise do ??
+exports.up = function (knex, Promise) {
 
     return Promise.all([
         knex.schema.createTable('food-available', (table) => {
+
             table.increments('id').primary();
             table.string('desc');
-            table.json('allergen');  // JSON should store it as just a string
+            table.json('allergen'); //It is a JSON object to be able to use the whereJsonNotSupersetOf Method in findFoodHandler
             table.float('lat');
             table.float('long');
-            table.string('address');
-            table.specificType('coord', 'point')
-            // how should I store the image ?
+            // table.string('address');  // I would still need to connect it to a google API to get the address from a lat, long
+            table.specificType('coord', 'point'); //Debating removing this because I dont end up using it, because of the bounding box with lat/long method
         })
     ]);
 };
 
-exports.down = function(knex, Promise) {
-
+exports.down = function (knex, Promise) {
+    //Not sure if I should delete this or keep it
```

Comment:
In the exports.down, you undo the changes in exports.up. So you'd drop the `food-available` table here